yarn-project/end-to-end/src/bench/client_flows/utils.ts

```diff
@@ -0,0 +1,267 @@
+import type {
+  ContractFunctionInteraction,
+  DeployMethod,
+  DeployOptions,
```

Comment:
Charlie has convinced me utils is a bad name and I can't unsee that it basically is "stuff.ts" :)